### PR TITLE
Adding Terminal Slope axis registry

### PR DESCRIPTION
**Pull request by vv-monsalve**
This axis modifies the slope and sharpness of stem terminals.
The inclusion of it was discussed in https://github.com/google/fonts/issues/2849

### CODE REVIEW

**Comment by vv-monsalve** [comment]
Proposal updated according to this [comment](https://github.com/google/fonts/issues/2849#issuecomment-738939428) in #2849

**Comment by RosaWagner** [comment]
Is it the final proposition?

File: axisregistry/terminal_slope.textproto [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#SHARP based on https://github.com/monokromskriftforlag/geologisk/#variable-axis
```
Comment (m4rc1e) [inline axisregistry/terminal_slope.textproto:1]: Change `SHARP` to `TSLO`.

File: axisregistry/terminal_slope.textproto [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  name: "Flat"
+  value: 0.0
+}
+fallback {
+  name: "Sloped"
+  value: 100.0
+}
+description: "Detailing and terminal treatment."
+  "Modifies the slope of the sans serif stem terminals from flat to sloped"
+  "As the slop increase, different levels of acuteness can be set"
```
Comment (m4rc1e) [inline axisregistry/terminal_slope.textproto:18]: Perhaps the description could simply be "Description: Adjust the slope angle of sans-serif stem terminals."?

File: axisregistry/terminal_slope.textproto (hunk 1 above)
Comment (vv-monsalve) [reply axisregistry/terminal_slope.textproto:1]: > Is it the final proposition?

No, I haven't updated it with the latest changes. I was waiting for final comments on the issue where it was discussed.
If we agree on the [latest proposal](https://github.com/google/fonts/issues/2849#issuecomment-739977724), and we don't need any further specific approval I would update it, including the suggestion on the description.

File: axisregistry/terminal_slope.textproto (hunk 2 above)
Comment (vv-monsalve) [reply axisregistry/terminal_slope.textproto]: Done

File: axisregistry/terminal_slope.textproto (hunk 1 above)
Comment (vv-monsalve) [reply axisregistry/terminal_slope.textproto:1]: I've just updated it

**Comment by vv-monsalve** [comment]
We are giving this a second revision to make it broader or "universal" like `Terminal Shape` so that it opens the possibility to include other possible shapes or forms in the future, e.g. rounded. 

So, I'm closing this here for now.

**Pull request closed without merging**